UKHomeOffice/cerberus-service
# COP-10891: Add tests to check No rules & selectors in a task

## Description
Add tests to check No rules & selectors in a task

## To Test
npm run cypress:runner

run test `Should verify No selectors & rules on task details page` from spec `airpax-task-details.spec.js`

## Developer Checklist

\* Required
